app/lib/actions/katello/content_view/refresh_rolling_repo.rb

```diff
@@ -0,0 +1,30 @@
+module Actions
+  module Katello
+    module ContentView
+      class RefreshRollingRepo < Actions::EntryAction
+        def plan(repository)
+          action_subject repository
+          sequence do
+            plan_self(repository_id: repository.id)
+            plan_action(Pulp3::Repository::RefreshDistribution, repository, SmartProxy.pulp_primary)
```

Comment:
There's an issue here with container push repositories. They're a bit special because, in Pulp, they cannot be redistributed.
We have special logic in `app/lib/actions/pulp3/orchestration/repository/copy_all_units.rb` that tells our orchestration to copy all of the units of a container push repo at publish time from the library repo to the content view version repo.

So, we have two options here. Either we actually copy content from the container push repositories, or we disallow container push repositories from rolling CVs.

Rolling CVs seem to mostly target content types that use sub-man (yum/deb), so I'd be okay with just disallowing container push repositories for now. Copying content would likely over-complicate this new CV type.

Reply:
In that case we definitely disallow container push repositories for now. Actually copying content very much breaks the Idea of keeping rolling CVs lightweight, which is an essential part of the feature, since we need "Add/Remove repo" operations to be fast.

Reply:
I missed your previous comment about using the rolling CV field, let me read that first.

Reply:
There are container push repositories for the Default org view, that's where the go first once you push them. We save references to the distributions that Pulp creates automatically.